### ISSUE TITLE
Switch to AWS

### ISSUE THREAD

**Issue by masevak** [laanuk/memora-public#3]
Now that we have AWS credit we should move over to hosting there instead of github pages.

**Comment by kurtcarpenter** [comment]
I thought we were using the memora-v2 repo. @masevak @supersam654

**Comment by masevak** [comment]
Yeah, we are but it'll probably be some time before we get that together and ready to go so for the time being we can transfer this over to AWS and then when that one is ready we can replace this one out w that (idk technically how to switch but y'all probably know what I mean).

**Comment by supersam654** [comment]
Getting you guys to update the marketing site is a bit easier if we stick with GitHub pages. I'll investigate making AWS deployments easy for you guys but I think that's a lower priority right now.
